infra/examples/README.md

```diff
@@ -11,13 +11,51 @@ files specific to your Terraform configuration (such as `terraform.tfvars`, `.te
 you opt for [local state](https://www.terraform.io/language/settings/backends/local), your
 `terraform.tfstate*` files) will be under version control.
 
-NOTE: for production use, we suggest you change the Terraform configuration to use a remote backend
-rather than your local file system, as some values serialized into your Terraform state should be
-handled as sensitive data.
-
 Professional services to assist in building a Terraform configuration appropriate for your needs
 is available from Worklytics. Please contact [sales@worklytics.co](mailto:sales@worklytics.co).
 
-
-
-
+## Dependencies
+
+These example configurations reference Psoxy-provided Terraform modules as dependencies via their
+remote addresses hosted in GitHub. If you prefer to modify any of these modules, you can convert
+these to local references throughout your configuration:
+  - replacing `git::https://github.com/worklytics/psoxy//infra/modules` with `../../modules`
```

Comment:
```suggestion
  - replacing `git::https://github.com/worklytics/psoxy/infra/modules` with `../../modules`
```
Double slash OK?

Reply:
required, in fact. Seems to be how it knows it's a path within a repo, rather than to a repo